pkg/kubelet/apis/pluginregistration/v1alpha/api.proto

```diff
@@ -0,0 +1,37 @@
+// To regenerate api.pb.go run hack/update-generated-kubelet-plugin-registration.sh
+syntax = 'proto3';
+
+package pluginregistration;
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+
+option (gogoproto.goproto_stringer_all) = false;
+option (gogoproto.stringer_all) =  true;
+option (gogoproto.goproto_getters_all) = true;
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.sizer_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+option (gogoproto.goproto_unrecognized_all) = false;
+
+message InfoResponse {
+	// Type of the Plugin. CSIPlugin or DevicePlugin
+	string type = 1;
+	// Plugin name.
+	string name = 2;
+        // Plugin service API version
+	string version = 3;
```

Comment:
+ @saad-ali @vishh @thockin 
I think we may want to make this a repeated fields that list all the supported service API versions from the plugin. From some quick experiments, I think you can create multiple versions of service on a single socket and we can make multiple clients on any of the supported version. That is probably the model we would like to advocate if a plugin needs to support multiple API versions for backward compatibility.

Reply:
Right. Different versions of services will be same as different services and we can run multiple services on the same socket. Stub device plugin in this PR is modified to run Identity service and main service on the same socket.